config/consensus.go

```diff
@@ -792,7 +792,7 @@ func initConsensusProtocols() {
 	vFuture.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
 
 	// FilterTimeout 2s instead of 4s for period 0
-	vFuture.FilterTimeoutPeriod0SmallLambdas = 1
+	vFuture.AgreementFilterTimeoutPeriod0 = 2 * time.Second
```

Comment:
for now, set it to `4 * time.Seconds` and update the comment that we should revisit this value.